samples/bluetooth/llpm/src/main.c

```diff
@@ -217,6 +217,7 @@ static int enable_llpm_mode(void)
 				sizeof(*cmd_enable));
 	if (!buf) {
 		printk("Could not allocate LLPM command buffer\n");
+		return -ENOMEM;
```

Comment:
While you are at it.
Could you please update the follow pattern in this file:

```
err = func()
if (err) {
  printk("Fail %d", err);
} else {
  printk("Success");
}
return err;
```
To
```
err = func()
if (err) {
  printk("Fail %d", err)
  return err;
}

printk("success");
return 0;
```